resource-managers/cook/src/main/scala/org/apache/spark/scheduler/cluster/cook/CoarseCookSchedulerBackend.scala

```diff
@@ -501,8 +497,10 @@ class CoarseCookSchedulerBackend(
         }
 
         Try[Unit](
-          jobClient
-            .submit(executorIdAndJob.map(_._2).asJava, jobListener)) match {
+          if (schedulerContext.cookPoolOption.isEmpty)
+            jobClient.submit(executorIdAndJob.map(_._2).asJava, jobListener)
+          else
+            jobClient.submit(executorIdAndJob.map(_._2).asJava, schedulerContext.cookPoolOption.get, Listener)) match {
```

Comment:
Why does this line have `Listener` instead of `jobListener`?

Reply:
good catch